app/views/spree/taxons/_introduction.html.erb

```diff
@@ -7,8 +7,10 @@
                <%= @taxon.name %>
             </h1>
             <div class="category-banner phone-hide">
-            <% if @taxon.icon&.attachment %>
+            <% if @taxon.icon&.attachment.present? %>
```

Comment:
please use this method : attached? for images. As its default active storage method for image presence. Please replace present? with attached? and confirm its working fine